modules/cnf-creating-nrop-cr-hosted-control-plane.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-creating-nrop-cr-hosted-control-plane_{context}"]
+= Creating the NUMAResourcesOperator custom resource for the hosted control plane
+
+When you have installed the NUMA Resources Operator, then create the `NUMAResourcesOperator` custom resource (CR) that instructs the NUMA Resources Operator to install all the cluster infrastructure needed to support the NUMA-aware scheduler on hosted control plane, including daemon sets and APIs.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NUMA Resources Operator.
+
+.Procedure
+
+.  Export the management cluster kubeconfig:
++
+[source,terminal]
+----
+$ export KUBECONFIG=</path/to/management-cluster-kubeconfig>
+----
+
+. Run this command to find the `node-pool-name` for your cluster:
++
+[source,terminal]
+----
+$ oc --kubeconfig="$MGMT_KUBECONFIG" get np -n A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE   NAME     CLUSTER   DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION   UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+clusters    europa   europa    2               2               False         False        4.18.5    False             False
+----
+. Create the `NUMAResourcesOperator` custom resource:
+
+.. Save the following minimal required YAML file example as `nrop-hcp.yaml`:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+  - poolName: europa <1>
+----
++
+<1> The `poolName` is the `node-pool-name` retrieved in step 2.
+
+.. Create the `NUMAResourcesOperator` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f nrop-hcp.yaml
+----
+
+.Verification
+
+. Verify that the NUMA Resources Operator deployed successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get numaresourcesoperators.nodetopology.openshift.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                    AGE
+numaresourcesoperator   27s
+----
+
+. After a few minutes, run the following command to verify that the required resources deployed successfully:
```

Comment:
let's also mention that the creation of the `numaresourcesscheduler` is similar as in standard OCP and it should be created on the hosted-cluster as well